src/index.js

```diff
@@ -1,5 +1,57 @@
 /* eslint-disable no-unused-vars */
-import _ from 'lodash';
+import _, { get } from 'lodash';
 import 'bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+const requestUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/0pCiglrVtP9Y26ppM44Z/scores/';
+const scoreBoard = document.querySelector('#scoresContainer');
+const refreshButton = document.querySelector('#refreshButton');
+const userName = document.querySelector('#userName');
+const userScore = document.querySelector('#userScore');
+const submitButton = document.querySelector('#submitScoreButton');
+
+async function newScore(user, score) {
+  const response = await fetch(requestUrl, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user,
+      score,
+    }),
+  });
+  return response.json();
+}
```

Comment:
- [ ] Use an arrow function instead of the function keyword for all your functions. This is part of the ES6 syntax.